resources/components/Navbar.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Button from "@mui/material/Button";
+import IconButton from "@mui/material/IconButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import { Menu, MenuItem, Paper } from "@mui/material";
+import { Link, generatePath } from "react-router-dom";
+
+const ButtonAppBar = () => {
+    const [anchorEl, setAnchorEl] = useState(null);
+    const open = Boolean(anchorEl);
+    const handleClick = (event) => {
+        setAnchorEl(event.currentTarget);
+    };
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    return (
+        <Box sx={{ flexGrow: 1 }}>
+            <AppBar position="static">
+                <Toolbar
+                    style={{ display: "flex", justifyContent: "space-between" }}
+                >
+                    <IconButton
+                        size="large"
+                        edge="start"
+                        color="inherit"
+                        aria-label="menu"
+                        sx={{ mr: 2 }}
+                        id="basic-button"
+                        aria-controls={open ? "basic-menu" : undefined}
+                        aria-haspopup="true"
+                        aria-expanded={open ? "true" : undefined}
+                        onClick={handleClick}
+                    >
+                        <MenuIcon />
+                    </IconButton>
+                    <Button color="inherit">Login</Button>
```

Comment:
Je me demande si ce color="inherit" est obligatoire ?

Reply:
code is taken from the website of material UI, if you delete `color="inherit"` button is no longer visible